galaxy/api/views/repository.py

```diff
@@ -220,12 +231,16 @@ def update(self, request, *args, **kwargs):
         if not request.user.repositories.filter(pk=instance.pk):
             request.user.repositories.add(instance)
 
-        import_task = tasks.create_import_task(repository, request.user)
-
         serializer = self.get_serializer(instance=instance)
         data = serializer.data
-        data['summary_fields']['latest_import'] = \
-            serializers.ImportTaskSerializer(import_task).data
+
+        # Don't create an import task if we're just updating the deprication
+        # status
+        if updated != ['deprecated']:
```

Comment:
Do we really need to create import task if other fields (e.g. `original_name`, `description`, etc.) got updated?

Reply:
Which fields should kick off an import if they are update?

Reply:
Technically we don't need to run an import when a repo get's updated. However, it doesn't hurt anything, if it does run. Currently in the UI, it's not possible for a users to update repository attributes, so I think we can leave what David has for now and revisit later, if needed.